src/components/Dropdown/Dropdown.tsx

```diff
@@ -23,17 +24,52 @@ interface RootProps {
 function Dropdown({
   open,
   onOpenChange,
-  side,
-  align,
+  align = "center", //Default value
+  side = "bottom", //Default value
   overlay,
   children,
   className,
   style,
+  arrow
 }: RootProps) {
   let classes = [DropdownStyles['sbui-dropdown__content']]
   if (className) {
     classes.push(className)
   }
+
+  /**
+   * Border radius classnames if arrow is enabled
+   */
+  let dropdownContentBorderClasses = []
+  if(!arrow) {
+    dropdownContentBorderClasses.push('rounded')
+  } else {
+    if(align === "start") {
+      if(side === "right") {
```

Comment:
I'm not sure this works using the prop values `align` and `side`

I think the only time this would break is if the Dropdown has any collision, and the Dropdown tries to reposition to another side, or align elsewhere.

If you 'offset' the arrow slightly you probably don't need to adjust the border radius and just avoid it completely. Should still look good :D 
[Arrow API ref](https://radix-ui.com/primitives/docs/components/dropdown-menu#arrow)
```js
<RadixDropdown.Arrow 
offset={8}
 // ..
/>

```

Reply:
This is actually a far better way, sad I missed it :(